components/organisms/AboutSection/about-section.styled.tsx

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components';
+
+import { FramedImage } from '@components/atoms';
+
+export const StyledAboutSection = styled.section`
+    display: grid;
+    grid-template-columns: 60% 40%;
+    grid-template-rows: 15% 45% 20% 20%;
+    gap: 3px 3px;
+    grid-template-areas:
+        'blank header'
+        'text image'
+        'more-link image'
+        'footer image';
+
+    max-width: 1170px;
+`;
+
+export const StyledH2 = styled.h2`
+    margin: 0;
```

Comment:
So one of the reason's you're needing to do margin: 0 is because of my global theme from my website which adds some margin in.

I recommend you clean up these global styles to better suit your own site.

You can find it in `styles/GlobalStyles.tsx`

Reply:
I possibly just left those in so there was something in all the styles i think.